File: CS/GroupSummaries/Models/Order.cs
using System.ComponentModel.DataAnnotations;

namespace GroupSummaries.Models
{
    public class Order
    {
        public int OrderID { get; set; }
        [Required]
        public string ProductName { get; set; }
        public int Price { get; set; }
        public int Quantity { get; set; }
        public DateTime SomeDate { get; set; }
        public string Group1 { get; set; }
        public string Group2 { get; set; }
        public bool Discontinued { get; set; }
    }
}

File: VB/GroupSummaries/Models/Order.vb
Imports System.ComponentModel.DataAnnotations

Namespace GroupSummaries.Models

    Public Class Order

        Public Property OrderID As Integer

        <Required>
        Public Property ProductName As String

        Public Property Price As Integer

        Public Property Quantity As Integer

        Public Property SomeDate As DateTime

        Public Property Group1 As String

        Public Property Group2 As String

        Public Property Discontinued As Boolean
    End Class
End Namespace
